docs/application-programming-model.md

```diff
@@ -11,7 +11,7 @@ Read/Write operations are done over the network. An operation may fail at any po
 
 To address the above concerns, We designed Waltz based on a stream oriented communication instead of a RPC based communication.
 
-![RPC communication design](/img/docs/rpc-comm.png)
+![RPC communication design](/waltz/img/docs/rpc-comm.png)
```

Comment:
Is this change going to work locally, or will images now not be visible locally?

Reply:
Do you mean, from the website published to localhost?

Reply:
Ah! I see what you meant.

Reply:
These pages https://github.com/wepay/waltz/tree/master/docs ?

(Looks like it is not working on current master, as Biplap pointed out. Let me update the directory structure.)